From 8894bbbfb2333ec812b51221ed6be88105d6e1d5 Mon Sep 17 00:00:00 2001
From: "Huang, Tao" <huangtao@rock-chips.com>
Date: Tue, 14 Mar 2017 14:44:22 +0800
Subject: [PATCH] net: wireless: rockchip: disable build rtl8822be pcie wifi by
 default

Until fix this compile error:
ERROR: "Array_MP_8822B_FW_NIC" [drivers/net/wireless/rockchip_wlan/rtl8822be/8822be.ko] undefined!

Change-Id: I78afda57761bb0bd7737864e5d04deeaa1b7df8a
Signed-off-by: Huang, Tao <huangtao@rock-chips.com>
---
 drivers/net/wireless/rockchip_wlan/rtl8822be/Kconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/net/wireless/rockchip_wlan/rtl8822be/Kconfig b/drivers/net/wireless/rockchip_wlan/rtl8822be/Kconfig
index 4d5967f60978..9cc4c2bd7792 100644
--- a/drivers/net/wireless/rockchip_wlan/rtl8822be/Kconfig
+++ b/drivers/net/wireless/rockchip_wlan/rtl8822be/Kconfig
@@ -1,7 +1,6 @@
 config RTL8822BE
 	tristate "Realtek 8822B PCIE WiFi"
 	depends on PCI
-	default m
 	---help---
 	  Help message of RTL8822BE
 
-- 
2.35.3

